doc/java-devbox-setup.md

```diff
@@ -89,14 +87,12 @@ You can verify that the environment variables necessary to run **_Maven 3_** hav
 ```
 mvn --version
 ```
-<a name="installiot"></a>
```

Comment:
I'm not sure if these were done wrong, but I have seen this style of labeling in markdown before. Intellij didn't agree with them though, so I removed them and linked to the heading title instead